### PR TITLE
fix(ux): Align filters horizontally on `md` breakpoints and higher

### LINKED ISSUE #5231: `/actors` - stack table filter controls horizontally on larger screens

### LINKED ISSUE #5232: `/groups` - stack filter controls horizontally on larger screens

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes #5231 
Fixes #5232 

Screenshot 2024-06-05 at 7 26 19 PM
Screenshot 2024-06-05 at 7 26 11 PM
Screenshot 2024-06-05 at 7 26 03 PM

Screenshot 2024-06-05 at 7 32 16 PM
Screenshot 2024-06-05 at 7 32 07 PM
Screenshot 2024-06-05 at 7 31 50 PM
Screenshot 2024-06-05 at 7 31 48 PM
Screenshot 2024-06-05 at 7 31 43 PM

### CODE REVIEW

File: elixir/apps/web/lib/web/components/form_components.ex [hunk 1]
```diff
@@ -237,54 +236,6 @@ defmodule Web.FormComponents do
     """
   end
 
-  def input(%{type: "taglist"} = assigns) do
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/form_components.ex:240]: This seemed like dead code

File: elixir/apps/web/lib/web/components/navigation_components.ex [hunk 2]
```diff
@@ -312,7 +312,7 @@ defmodule Web.NavigationComponents do
     <div class="mt-16">
       <.link
         navigate={@navigate}
-        class="text-sm font-semibold leading-6 text-zinc-900 hover:text-zinc-700"
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/navigation_components.ex:315]: Not sure if we were intentionally leaving `zinc` in a few places?

File: elixir/apps/web/lib/web/live/groups/show.ex [hunk 3]
```diff
@@ -111,8 +111,7 @@ defmodule Web.Groups.Show do
           }
           kind={:info}
         >
-          <p>This group is managed by Firezone and cannot be edited.</p>
-          <p>It will contain all actors with at least one authentication identity.</p>
+          <p>This Group contains all Users and cannot be edited.</p>
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/live/groups/show.ex:114]: Tried to make this slightly simpler to understand.

**Review by AndrewDryga — APPROVED** [review]

**Comment by AndrewDryga** [comment]
Code-wise LGTM. But the design part is controversial.. this looks good:

Screenshot 2024-06-06 at 12 37 16

while this one (to me personally) looks much worse:
Screenshot 2024-06-06 at 12 38 28
Screenshot 2024-06-06 at 12 38 23

**Comment by jamilbk** [comment]
> Code-wise LGTM. But the design part is controversial.. this looks good:
> 
> Screenshot 2024-06-06 at 12 37 16
> while this one (to me personally) looks much worse: Screenshot 2024-06-06 at 12 38 28 Screenshot 2024-06-06 at 12 38 23

Yeah they could be less wide. I'll remove the `w-1/3` and they'll take up less needless horizontal soace.

**Comment by jamilbk** [comment]
The longterm fix for many filters would be a dropdown or popover so they're out of the way, but that would take a few more hours

**Comment by jamilbk** [comment]
Screenshot 2024-06-06 at 12 42 30 PM

**Pull request merged**